package.json

```diff
@@ -1,38 +1,31 @@
 {
-  "name": "webpack-boilerplate",
-  "version": "1.0.0",
-  "description": "React / Webpack / Babel framework for quick set up",
-  "main": "index.js",
+  "name": "athenaeum",
+  "version": "0.0.4",
+  "description": "PolicyGenius React Component Library",
+  "main": "lib/index.js",
   "scripts": {
     "open": "open http://localhost:6060",
     "c": "yarn run plop create-component",
-    "clean": "rm -rf ./dist/*",
-    "build": "webpack --config ./webpack.config.js --progress --colors",
-    "build:prod": "NODE_ENV='production' webpack --config ./webpack.config.js --progress --colors",
-    "start": "node ./app.js",
-    "dev": "webpack-dev-server --inline --hot",
+    "build": "./node_modules/webpack/bin/webpack.js --config ./webpack.config.dev.js --progress --colors",
+    "build:prod": "./node_modules/webpack/bin/webpack.js --config ./webpack.config.prod.js --progress --colors",
+    "publish": "yarn run publish",
+    "start": "yarn run styleguide:build && yarn run styleguide",
     "test": "karma start",
     "stylin": "yarn run styleguide:build && yarn run styleguide",
```

Comment:
ah, yeah. we might want to just stick with start since that is a pretty typical convention